tools/pre-commit.sh

```diff
@@ -2,6 +2,5 @@
 
 set -e
 
-export DOCKER_BUILDKIT=1
```

Comment:
This was done to avoid long lines. It makes it more readable.

Reply:
I see. Just wanna be consistent wiht `run_sanity_check.sh`, that's why I remove it. I'll add it back. Thanks @gabrieldemarmiesse.